docs/modules/ROOT/pages/capacity-planning.adoc

```diff
@@ -331,13 +346,14 @@ reads the data from the sources and writes the results to the sinks. The
 capacity of all components of the data pipeline must be balanced to
 avoid bottlenecks.
 
-If a data sink is slow, Hazelcast applies backpressure all the way to the
+If a data sink is slow, Hazelcast applies xref:cluster-performance:best-practices.adoc#back-pressure[backpressure] 
+all the way to the
 source, slowing down data consumption. The data sources should be
 designed to participate by reducing the pace of data production or by
 buffering the data.
 
-On the other hand, if the data source can't produce or transmit the
-data fast enough, adding more resources to the Hazelcast cluster won't bring
+On the other hand, if the data source is slow (i.e. it can't produce or transmit the
```

Comment:
... is slow, i.e., it can't .....fast enough, adding more ...

Reply:
Changed